proto/kaskada/kaskada/v1alpha/common.proto

```diff
@@ -24,6 +25,8 @@ message SourceData {
     string csv_data = 3;
 
     PulsarSubscription pulsar_subscription = 4;
+
+    KafkaSubscription kafka_subscription = 5;
```

Comment:
it is a bit odd that we need a `Subscription` message for the `getMetadata()` call.  

I think this is because the `SourceData` message is used for both `getMetadata()` and `prepareFile()`

Maybe instead of adding the KafkaSubscription here, we change `GetMetadataRequest` to be a one-of `SourceData`, `PulsarConfig`, or `KafkaConfig`?

Reply:
thoughts on this @kevinjnguyen ?

Reply:
I added this in the subsequent changes.